applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -303,6 +303,7 @@ def _add_variables_to_model_part(self, model_part):
         model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PRESSURE)
         model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VOLUME_ACCELERATION)
         model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_MASS)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_AREA)
```

Comment:
do you really need this to be historical?

Reply:
@philbucher I plan to make it changeable while the coupling with the structure is performed, i.e. when a surface is stretched in case of a membrane, it has to have more weight.

Reply:
sure it should be changeable, but do you need the history for that?

Reply:
I got it! No need this, you are right